src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/EditPersonModal/EditPersonModal.tsx

```diff
@@ -44,80 +50,262 @@ const ShowExtraText = styled(Typography)(() => ({
   fontWeight: 'bold',
 }));
 
+const LoadingIndicator = styled(CircularProgress)(({ theme }) => ({
+  margin: theme.spacing(0, 1, 0, 0),
+}));
+
+const phoneRegex = RegExp(/^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/);
+
+const personSchema: yup.SchemaOf<
+  Omit<PersonUpdateInput, 'familyRelationships' | 'id'>
+> = yup.object({
+  firstName: yup.string().required(),
+  lastName: yup.string().required(),
+  title: yup.string().nullable(),
+  suffix: yup.string().nullable(),
+  phoneNumbers: yup.array().of(
+    yup.object({
+      id: yup.string().nullable(),
+      number: yup
+        .string()
+        .matches(phoneRegex, 'Invalid phone number')
+        .required('This field is required'),
```

Comment:
Translate

Reply:
I also have the translate here: https://github.com/CruGlobal/mpdx-react/blob/27b662fe4858061a2c53b423e999a4ca2474eaf7/src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/EditPersonModal/PersonPhoneNumber/PersonPhoneNumber.tsx#L120